### PR TITLE
display single posts at their slug and show all posts on index

### PR DESCRIPTION
**Pull request by jeremystephencobb**
- load posts at `/posts/:slug`

- load all posts on the homepage

- copied the errorHandlers from the WB app and am using `catchErrors` on async / await functions

Completes #6

### CODE REVIEW

File: views/mixins/_postItem.pug [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//- this is basically our 'grid-item' for posts
+mixin postItem(post = {})
+  .post
+    .post--title
```
Comment (andrewmcgov) [inline views/mixins/_postItem.pug]: We should add a link to the post here just to make it easy for us to navigate for now (I know you just slung this together to make sure it works haha)

File: views/singlePost.pug [hunk 2]
```diff
@@ -0,0 +1,6 @@
+extends layout
+
+block content
+  h2= title
+  .content.mce
```
Comment (andrewmcgov) [inline views/singlePost.pug]: Do we need the `.mce` class here if we are just rendering the content and not editing it in the RTE?

File: views/mixins/_postItem.pug (hunk 1 above)
Comment (jeremystephencobb) [reply views/mixins/_postItem.pug]: done

**Comment by jeremystephencobb** [comment]
Good feedback, I actioned on it and merged.

**Pull request merged**